libstorage/BinLogHandler.cpp

```diff
@@ -99,4 +101,123 @@ bool BinLogHandler::getMissingBlocksFromBinLog(
     return true;
 }
 
-void BinLogHandler::encodeBlock(int64_t, const std::vector<TableData::Ptr>&, bytes&) {}
+void BinLogHandler::writeUINT32(bytes& buffer, uint32_t ui)
+{
+    uint32_t ui_tmp = htonl(ui);
+    buffer.insert(buffer.end(), (byte*)&ui_tmp, (byte*)&ui_tmp + 4);
+}
+
+void BinLogHandler::writeUINT64(bytes& buffer, uint64_t ui)
+{
+    writeUINT32(buffer, ((ui >> 32) & 0xffffffff));
+    writeUINT32(buffer, (ui & 0xffffffff));
```

Comment:
wrong impl

Reply:
Now it is processed separately according to the situation of the machine.